src/better_lane_follow/test/my_node_rostest.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+ * Created By: Valerian Ratu
+ * Created On: January 29, 2017
+ * Description: Integration testing for MyNode
+ */
```

Comment:
Not sure if we're going to have time for rostests along with the unit tests, so let's just remove this file for now. I think it's just copied over from the `IGVC-2017` repo, so we should be able to easily re-add it if needed.